scripts/irods/test/test_igroupadmin.py

```diff
@@ -72,6 +71,63 @@ def tearDownClass(self):
             self.admin.__exit__()
             admin_session.assert_icommand(['iadmin', 'rmuser', self.admin.username])
 
+    def test_create_user_with_password_in_igroupadmin__6887(self):
+        USERS = { 'alice':42,
+                  'bob':40, 'eve': 3 }
+
+        # TODO -- eventually delete , but using for test r.n.
+        #obj_name = '{}/{}'.format(self.groupadmin.home_collection, 'test__6887__object')
+        #self.groupadmin.assert_icommand(['ichmod', 'write', self.groupadmin.username, obj_name], 'EMPTY' )
+
+        for username,pwlen in USERS.items():
+            try:
+                passwd = (username[0]+'pass_6887_the_quick_brown_fox_jumps_over_the_lazy_dog')[:pwlen]
+                self.groupadmin.assert_icommand(['igroupadmin', 'mkuser', username, passwd], 'EMPTY' )
+                with session.make_session_for_existing_user(username, passwd, lib.get_hostname(), self.admin.zone_name):
+                    pass
```

Comment:
Add a comment explaining why these lines are needed.

Reply:
Yeah, sorry. temporarily needed only. They'll be gone soon. I was exploring the `make_session_for_existing_user` thing as I hadn't used it before.

Reply:
Got it.

Reply:
Or maybe, needed.  Just to ensure the password works for the login. Helps to issue an icommand as well.  So now that is in final form, I think.